### PR TITLE
Remove `ms` dependency

### PR DESCRIPTION
**Pull request by Bamieh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "hapi-static-files",
-  "version": "1.3.0",
```
Comment (rubennorte) [inline package.json]: Could you please leave the version number unmodified so we can bump it in the next release?

**Comment by rubennorte** [comment]
Thanks for the PR. The `ms` package isn't necessary in this case indeed. We used it because this was part of our application and we didn't refactor it after we extracted the package.

**Comment by Bamieh** [comment]
You're welcome, i've reverted the version number.

**Comment by rubennorte** [comment]
Released as v1.3.1. Thanks for your contribution!

**Pull request merged**